example/service.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import time
+import requests
+
+nitriding_url = "http://127.0.0.1:8080/enclave/ready"
+
+
+def signal_ready():
+    ok = 200
+    r = requests.get(url=nitriding_url)
+    if r.status_code != ok:
+        raise Exception("Expected status code %d but got %d" %
+                        (ok, r.status_code))
+
+
+def fetch_addr():
+    r = requests.get(url="https://ifconfig.me/ip")
+    print("[py] Our IP address is: %s" % r.text)
```

Comment:
I would `r.raise_for_status()` here though. It isn't much complication and an error logging path can be helpful when exploring examples.

Reply:
Fixed in 41d8af8.